### ISSUE TITLE
Revisit page titles to match page H1s

### ISSUE THREAD

**Issue by anikbrazeau** [cds-snc/notification-api#979]
The page title in the web browser tab is different than the H1 of the page, and some are missing French translations

For example:

Security is in English only
https://notification.alpha.canada.ca/features/security

Privacy does not match the H1 on the page
https://notification.alpha.canada.ca/privacy

Revisit pages, as others may have the same issue.
This should be consistent with the heading on the page.
